components/lib/crash/src/main/java/mozilla/components/lib/crash/prompt/CrashReporterActivity.kt

```diff
@@ -40,6 +44,10 @@ class CrashReporterActivity : AppCompatActivity() {
 
         titleView.text = getString(R.string.mozac_lib_crash_dialog_title, appName)
         sendCheckbox.text = getString(R.string.mozac_lib_crash_dialog_checkbox, organizationName)
+
+        sharedPreferences = PreferenceManager.getDefaultSharedPreferences(this)
```

Comment:
We should avoid using the default shared preferences here: We do not want to interfere with the app. Let's use a different file here. Something like `mozac_lib_crash_settings.xml`.

Reply:
Could do please explain me what do I do with this file?